lib/cmd/fh3/app/undeploy.js

```diff
@@ -0,0 +1,48 @@
+var log = require("../../../utils/log");
+
+module.exports = {
+  'desc': 'Undeploy an app given the domain and environment it is deployed to',
+  'examples': [{
+    cmd: 'fhc app undeploy --id=<app-id> --env=<environment> --domain=<domain>',
+    desc: 'Undeploy <app-id> from environment <environment> in domain <domain>'
+  }],
+  'demand': ['id', 'env', 'domain'],
+  'alias': {
+    'id': 'i',
+    'env': 'e',
+    'domain': 'd',
+    0: 'id',
+    1: 'env',
+    2: 'domain'
```

Comment:
does domain require required? I think it is possible to get it from the current fhc target?